quarkus-integration/quarkus/deployment/src/test/java/ai/timefold/solver/quarkus/rest/TimefoldProcessorHotReloadTest.java

```diff
@@ -1,21 +1,32 @@
 package ai.timefold.solver.quarkus.rest;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 import ai.timefold.solver.quarkus.testdata.normal.constraints.TestdataQuarkusConstraintProvider;
 import ai.timefold.solver.quarkus.testdata.normal.domain.TestdataQuarkusEntity;
 import ai.timefold.solver.quarkus.testdata.normal.domain.TestdataQuarkusSolution;
 
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
-import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
 import io.quarkus.test.QuarkusDevModeTest;
 import io.restassured.RestAssured;
 
+/**
+ * Do not remove the public modifier from this class. Otherwise, the test will fail with the following:
```

Comment:
Dtto. I'm even considering if we need to warn about this at all; we typically don't.

Reply:
The comment was simplified. It took me some time to figure out the test was failing because of the missing modifier, and I think having it is a good idea.